inductiva/tasks/task.py

```diff
@@ -791,6 +793,34 @@ def _request_download_output_url(
 
         return api_response.body
 
+    def _request_download_input_url(
+        self
+    ) -> Optional[get_tasks_task_id_download_input_url.
+                  SchemaFor200ResponseBodyApplicationJson]:
+        try:
+            api_response = self._api.get_input_download_url(
+                path_params=self._get_path_params(),)
+        except exceptions.ApiException as e:
+            if not self._called_from_wait:
+
+                if self._status == models.TaskStatusCode.EXECUTERFAILED:
```

Comment:
Even if the executer fails don't we have the inputs to download ?

Reply:
Yes, I'll fix this. Thanks!